### DISCUSSION TITLE
Migrate from Jira to Github Issues and Projects

### DISCUSSION THREAD

**Discussion by arolfes** [Taskana/taskana#2118] [Ideas]
Hello @holgerhagen , @gitgoodjhe , @mustaphazorgati 

what was the outcome of the discussion from migrate the issues from JIRA into Github Issues and Github Projects?

**Comment by arolfes** [comment]
Hello @holgerhagen, @gitgoodjhe, @mustaphazorgati,

can you share your opinion on this topic?

My suggestion is:

After release 6.0.0
we migrate all open jira issues manually,
don't create new tickets in jira anymore. use github.
Closed tickets are exported as PDF and created as closed issue with the attached PDF.

What do you think?

**Reply by holgerhagen** [reply]
Hi @arolfes ,
in general I agree and we already decided this some time ago. We were just to busy to spend the effort to migrate manually. 
Wouldn't it be a good idea by starting with cleaning up our current Jira? We really have a huge number of tickets in there. And I'm not sure about the status of all of them.
As the backlog is the responsibility of the product owner we might need to discuss this first before we go on. At least this might make sense.

**Reply by arolfes** [reply]
Hi @holgerhagen ,

Then I make a decision :)

After the next Release 6 I go through All issues with Mustapha, we decide what to keep and what not

I will do the migration of the issues to github

With the next maintanance Release 6.1 only github issues will be resolved. I will take care that Jira project will be deleted.

**Comment by arolfes** [comment]
Hi @holgerhagen , @gitgoodjhe @mustaphazorgati ,

the migration is completed

**Reply by mustaphazorgati** [reply]
Awesome! :) 
Do we have tickets for our new processes?
Example:
* How to generate Release Notes
* Commit Guidelines (our hooks need to be adjusted)
* ...

**Comment by mustaphazorgati** [comment]
I have made our JIRA Boards read-only. Additionally, I made them public, so that they can be viewed from everyone.
